src/devices/PiJuice/Models/WakeUpOnCharge.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.PiJuiceDevice.Models
+{
+    /// <summary>
+    /// Wake up on charge configuration
+    /// </summary>
+    public class WakeUpOnCharge
+    {
+        /// <summary>
+        /// Is the wake up on charge disabled
+        /// </summary>
+        public bool Disabled { get; set; }
+
+        /// <summary>
+        /// Battery charge level percentage between [0 - 100] used to wake up the Raspberry Pi
+        /// </summary>
+        public short WakeUpPercentage { get; set; }
```

Comment:
I'm debating with myself for what is percent. For bindings with humidity and the rest, we use Ratio. I guess for consistency, would be better to use Ratio as well. Maybe @krwq have some opinion as well.